js/helper.js

```diff
@@ -4,42 +4,86 @@ const os = require('os');
 const path = require('path');
 const childProcess = require('child_process');
 
+/**
+ * Absolute path to the sentry-cli binary (platform dependant).
+ * @type {string}
+ */
 const binaryPath =
   os.platform() === 'win32'
     ? path.resolve(__dirname, '..\\bin\\sentry-cli.exe')
     : path.resolve(__dirname, '../sentry-cli');
 
+/**
+ * Converts the given option into a command line args array.
+ *
+ * The value can either be an array of values or a single value. The value(s) will be
+ * converted to string. If an array is given, the option name is repeated for each value.
+ *
+ * @example
+ * expect(transformOption('--foo', 'a'))
+ *   .toEqual(['--foo', 'a'])
+ *
+ * @example
+ * expect(transformOption('--foo', ['a', 'b']))
+ *   .toEqual(['--foo', 'a', '--foo', 'b']);
+ *
+ * @param {string} option The literal name of the option, including dashes.
+ * @param {any[]|any} values One or more values for this option.
+ * @returns {string[]} An arguments array that can be passed via command line.
+ */
 function transformOption(option, values) {
   if (Array.isArray(values)) {
-    return values.reduce((acc, value) => acc.concat([option.param, value]), []);
+    return values.reduce((acc, value) => acc.concat([option.param, String(value)]), []);
   }
 
-  return [option.param, values];
+  return [option.param, String(values)];
```

Comment:
The value is now forcefully converted to `String` here. This makes it a bit more deterministic and also debuggable in case incompatible values are passed in.